doc/source/mast_data_products/hap-parameters.rst

```diff
@@ -459,7 +495,7 @@ perform_fit (*primarily external in tweakwcs.matchutils.XYXYMatch*)
 For match_relative_fit, match_default_fit, and match_2dhist_fit, the following parameters are used:
 
 fitgeom": "rscale",
-    As used above, this is ignored for pipeline products. 
+    As used above, this is ignored for pipeline products.
```

Comment:
Since you are discussing three categories at once for brevity, I would not quote the "default" values as they are not the same for all categories.

Reply:
default values removed.